src/script/conversation/MessageRepository.ts

```diff
@@ -1259,15 +1259,31 @@ export class MessageRepository {
     this.logger.info(`Sent countly sync message with ID ${countlyId}`);
   }
 
+  /**
+   * Sends a call message only to self conversation (eg. REJECT message that warn the user's other clients that the call has been picked up)
+   * @param payload
+   * @returns
+   */
+  public sendSelfCallingMessage(payload: string, targetConversation: QualifiedId) {
+    return this.sendCallingMessage(this.conversationState.getSelfMLSConversation(), {
+      content: payload,
+      qualifiedConversationId: targetConversation,
+    });
+  }
   /**
    * Send call message in specified conversation.
    *
    * @param eventInfoEntity Event info to be send
    * @param conversationId id of the conversation to send call message to
    * @returns Resolves when the confirmation was sent
    */
-  public sendCallingMessage(conversation: Conversation, payload: string, options: MessageSendingOptions = {}) {
-    const message = MessageBuilder.buildCallMessage({content: payload});
+  public sendCallingMessage(
+    conversation: Conversation,
+    payload: string | {content: string; qualifiedConversationId: QualifiedId},
```

Comment:
sendSelfCallingMessage(payload: string)

But here we are passing string or object, why?

Reply:
See [here](https://github.com/wireapp/wire-webapp/pull/14329/files#diff-8a277356da1eb1435094433268c1e8cf7392c374f617174bd0a6e60ea52ea4ebR1268), now we need a more complex object under certain circumstances (namely when in an MLS conversation).